### DISCUSSION TITLE
How to tell which plot file found a block?

### DISCUSSION THREAD

**Discussion by number435398** [Chia-Network/chia-blockchain#7748] [Q&A]
I set my log file to "INFO" (which, btw, I really think should be separate from the debug.log file otherwise there's so much stuff to wade through to find errors). Then, after a block was found I looked into the log file and couldn't confirm which plot file found the block. Sometimes the log file claims its having problems with a particular plot file, but if that file later finds a block, then you know its good. So there is a use to this. It'd also be fun/nice to know.

**Comment by rmica** [comment]
#3309

**Comment by rmica** [comment]
It would be nice to have the information in the harvester's log after a found proof.

**Comment by keliew** [comment]
FWIW, there's a powershell script from a community member: https://github.com/MrPig91/PSChia/blob/main/PSChia/Public/Get-ChiaFarmedPlot.ps1

He has a bunch of scripts too @ https://github.com/MrPig91/PSChia

**Comment by number435398** [comment]
I appreciate the answers you guys have given, though I guess I was hoping that the way to do it wasn't so elaborate. I thought it would just straight up tell me in the log file.

**Reply by keliew** [reply]
Yeah, it's not that straightforward...

**Comment by SlugFiller** [comment]
The issue with all the suggested answers here and the linked discussion is that they only apply to successfully farmed blocks, and not to proof that did not result in a block (can occasionally happen), nor partial proof that resulted in points.
